src/DisplayBoard.cpp

```diff
@@ -0,0 +1,120 @@
+#include "DisplayBoard.hpp"
+
+namespace
+{
+    const char HORIZONTAL = '-';
+    const char VERTICAL = '|';
+    const char INTERSECTION = '+';
+    const char PADDING = ' ';
+
+
+    std::string drawPadding(const unsigned int width,
+                            const char sign = PADDING);
+
+    std::string drawEndLine(const unsigned int widthRows, const unsigned int  width = 0);
+    std::string drawEmptyLine(const unsigned int width,
+                              const char sign = PADDING);
+
+    std::string drawColumns(const unsigned int maxElementsInRows,
+                            const unsigned int colNumber,
+                            const unsigned int heightCol);
+
+    std::string drawRow(const unsigned int maxElementsInRows);
+    std::stringstream drawBoard(const unsigned int width,
+                                const unsigned int height,
+                                const unsigned int widthRows,
+                                const unsigned int heightCol);
+}
+
+namespace DisplayBoard
+{
+    std::string displayInterface(const Board& b){
```

Comment:
Do not indent functions after namespace scope 
The same below.

Reply:
done